google-account-plugin/src/com/google/gct/login/GoogleLogin.java

```diff
@@ -323,7 +323,7 @@ public void logIn() {
    * either succeeds or fails.
    */
   public void logIn(@Nullable final String message, @Nullable final IGoogleLoginCompletedCallback callback) {
-    UsageTrackerService.getInstance().trackEvent(LoginTracking.CATEGORY, LoginTracking.LOGIN, "login.start", null);
+    UsageTrackerProvider.getInstance().trackEvent(LoginTracking.CATEGORY, LoginTracking.LOGIN, "login.start", null);
```

Comment:
shouldn't we be using intellij's service factory methods to resolve the provider now that we've made it a service?